packages/headless/src/controllers/commerce/core/sub-controller/headless-sub-controller.ts

```diff
@@ -38,19 +38,49 @@ import {
 } from '../sort/headless-core-commerce-sort';
 
 export interface BaseSolutionTypeSubControllers {
-  interactiveProduct: (props: InteractiveProductProps) => InteractiveProduct;
-  pagination: (props?: PaginationProps) => Pagination;
+  /**
+   * Creates an instance of the InteractiveProduct sub-controller.
```

Comment:
For consistency with the below examples
```suggestion
   * Creates an `InteractiveProduct` sub-controller.
```